spec/app_board_view_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "spec_helper"
+
+RSpec.describe "Board view", type: :feature do
+  before do
+    Capybara.app = Sinatra::Application
+    stub_const("CACHE_EXPIRY_TIMEOUT", 0)
+  end
+
+  context "get '/board/dxw'" do
+    it "shows today's date" do
+      stub_gcal_request(gcal_fake_response([]))
+      visit "/board/dxw"
+
+      expect(page).to have_selector("#datetime_date", text: Date.today.strftime("%A, %B %e"))
+    end
+    it "shows the Zoom call rooms A, B, and C" do
+      stub_gcal_request(gcal_fake_response([]))
+      visit "/board/dxw"
+
+      expect(page).to have_content("Zoom Call A")
+      expect(page).to have_content("Zoom Call B")
+      expect(page).to have_content("Zoom Call C")
+    end
+  end
+
+  context "get '/board/hoxton" do
+    it "shows the three meeting rooms" do
+      stub_gcal_request(gcal_fake_response([]))
+      visit "/board/hoxton"
+
+      expect(page).to have_content("Main Meeting Room")
```

Comment:
Here you're correctly testing that the rooms you expect to see are there - it might be worth adding a test that rooms you don't expect to see aren't.

Reply:
I like the idea. I think a test like that would be very useful. I'll add them. Thanks!